src/components/study-container.js

```diff
@@ -277,6 +278,14 @@ export function StudyContainer({ view, onChangeTab }) {
                 networkModificationTreeModel.setTreeElements(tree);
                 networkModificationTreeModel.updateLayout();
 
+                fetchCaseInfos(studyUuid)
+                    .then((res) => {
+                        if (res) {
+                            networkModificationTreeModel.setCaseName(res);
+                        }
+                    })
+                    .catch((err) => console.log(err.message));
```

Comment:
use
`console.error`
instead ?

Reply:
i think we should give the user a better feedback if there is an error, (in snack-bar for example)